.gitignore

```diff
@@ -6,3 +6,6 @@ scratch
 .bak
 venv/
 *.swp
+.tox
+reports/
```

Comment:
we had this directory before, so let's not bring this back as this most likely is just a quick way to not clean up.